### DISCUSSION TITLE
Numpy

### DISCUSSION THREAD

**Discussion by mssanamateen** [mssanamateen/Artificial-Intelligence-Lab#23] [Q&A] [answered]
shape property is used for?

**Comment by Abdullah1607** [comment]
The shape property is used to set or return the value of the shape attribute of an area.

It also specifies attribute and the shape of an area.

**Comment by sainathch45** [comment, marked as answer]
The shape property in numpy library is used for returning the number of elements along each axis (i.e.), dimension. It is not a method but a property that is why it is called without any parenthesis.
